### PR TITLE
[IMP] maintain: process for odoo.sh to odoo online

### PR DESCRIPTION
**Pull request by jaae-odoo**
https://www.odoo.com/web#id=2870122&cids=1&model=project.task&view_type=form

### CODE REVIEW

File: content/administration/maintain/hosting_changes.rst [hunk 1]
```diff
@@ -48,13 +48,34 @@ From Odoo Online to Odoo.sh
...
+   - The URL you want to use for your database (e.g.: `example.odoo.com`)
+   - Which brand you want to migrate
+   - In which region you want to be hosted (select the one closest to your users to reduce latency):
+
+     - Americas
+     - Europe
+     - Asia
+
+   - Which user(s) will be the administrator(s) (they must have an odoo.com account)
+   - When (and in which timezone) you want the database to be up and running (see note below)
```
Comment (LoredanaLrpz) [inline content/administration/maintain/hosting_changes.rst:66]: I took off the parenthesis -just bc I don't like it here and think it looks better as a note- (same with see as note)
```suggestion
   - Which user(s) will be the administrator(s)
   - When (and in which timezone) you want the database to be up and running
```

File: content/administration/maintain/hosting_changes.rst [hunk 2]
```diff
@@ -48,13 +48,34 @@ From Odoo Online to Odoo.sh
...
 
-1.  Uninstall all the **non-standard apps**: test it in a staging build first, then do it in your production build.
-2.  **If you have time constraints, contact us earlier to schedule the transfer.**
-3. `Create a support ticket <https://www.odoo.com/help>`_ and include your subscription number and the URL you want to use for your database (e.g.: my-company.odoo.com).
-4.  We will make sure your database is compatible and upload it to our cloud. In case of technical issues, we will get in touch with you.
-5.  It's done!
+   - Your subscription number
+   - The URL you want to use for your database (e.g.: `example.odoo.com`)
+   - Which brand you want to migrate
+   - In which region you want to be hosted (select the one closest to your users to reduce latency):
```
Comment (LoredanaLrpz) [inline content/administration/maintain/hosting_changes.rst:59]: I took off the parenthesis and put it as a note below
```suggestion
   - In which region you want to be hosted:
```

File: content/administration/maintain/hosting_changes.rst [hunk 3]
```diff
@@ -48,13 +48,34 @@ From Odoo Online to Odoo.sh
 From Odoo.sh to Odoo Online
 ===========================
 
-.. warning:: Odoo Online is not compatible with **non-standard apps**.
+#.  Uninstall all the **non-standard apps**: test it in a staging build first, then do it in your
+    production build.
+#.  **If you have time constraints, contact us earlier to schedule the transfer.**
```
Comment (LoredanaLrpz) [inline content/administration/maintain/hosting_changes.rst:53]: + indentation problem

```suggestion
#. **If you have time constraints, contact us earlier to schedule the transfer.**
```

Also, I wouldn't put this as a mandatory step, but in the note instead.

>>>> I guess you just put it this way to stay consistent with the top of the page, though, so maybe just leave it, or change the whole page MOUHAHAHAHHAHA

**Review by LoredanaLrpz — APPROVED** [review]

File: content/administration/maintain/hosting_changes.rst [hunk 4]
```diff
@@ -48,13 +48,37 @@ From Odoo Online to Odoo.sh
...
+#. Uninstall all the **non-standard apps**
+#. `Create a support ticket <https://www.odoo.com/help>`_ and include the following:
 
-1.  Uninstall all the **non-standard apps**: test it in a staging build first, then do it in your production build.
-2.  **If you have time constraints, contact us earlier to schedule the transfer.**
-3. `Create a support ticket <https://www.odoo.com/help>`_ and include your subscription number and the URL you want to use for your database (e.g.: my-company.odoo.com).
-4.  We will make sure your database is compatible and upload it to our cloud. In case of technical issues, we will get in touch with you.
-5.  It's done!
+   - Your subscription number
+   - The URL you want to use for your database (e.g.: `example.odoo.com`)
```
Comment (jcs-odoo) [inline content/administration/maintain/hosting_changes.rst:55]:
```suggestion
   - The URL you want to use for your database (e.g., `example.odoo.com`)
```

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
@toaa-odoo @LoredanaLrpz 
Nicely done! It looks great :) 
Just some full stops missing

**Review by jcs-odoo — APPROVED** [review]

**Comment by Feyensv** [comment]
@robodoo r+

**Review by Feyensv — APPROVED** [review]

**Pull request closed without merging**